contributing_to_docs/doc_guidelines.adoc

```diff
@@ -266,41 +266,23 @@ Do not use special characters or symbols in titles. Symbols and special characte
 
 Use only one level 1 heading (`=`) in any file.
 
-=== Discrete headings
-
-If you have a section heading that you do not want to appear in the TOC (like if you think that some section is not worth showing up or if there are already too many nested levels), you can use a discrete (or floating) heading:
-
-https://docs.asciidoctor.org/asciidoc/latest/blocks/discrete-headings/
-
-A discrete heading also will not get a section number in the Customer Portal build of the doc. Previously, we would use plain bold mark-up around a heading like this, but discrete headings also allow you to ignore section nesting rules (like jumping from a `==` section level to a `====` level if you wanted for some style reason).
-
-To use a discrete heading, just add `[discrete]` to the line before your unique ID. For example:
-
-----
-[discrete]
-[id="managing-authorization-policies_{context}"]
-== Managing authorization policies
-----
-
-== Anchoring titles and section headings
+=== Anchoring titles and section headings
 
 All titles and section headings must have an anchor ID. The anchor ID must be similar to the title or section heading.
 
-=== Anchoring in assembly files
-
-The following is an example anchor ID in an assembly file:
+==== Anchoring in assembly files
 
+.Example anchor ID in an assembly file
 ----
 [id="configuring-alert-notifications"]
 = Configuring alert notifications
 ----
 
-[NOTE]
-====
 Do not include line spaces between the anchor ID and the section title.
-====
```

Comment:
Didn't seem like this needed to be a NOTE when sections right before it just had line-by-line "Do not...", "Do not...", etc. (i.e., this one didn't seem more special).